edb/server/metrics.py

```diff
@@ -199,3 +199,30 @@
     "Number of successful logins in the Auth extension.",
     labels=("tenant",),
 )
+
+mt_tenants_total = registry.new_gauge(
+    'mt_tenants_current',
+    'Total number of currently-registered tenants.',
+)
+
+mt_config_reloads = registry.new_counter(
+    'mt_config_reloads_total',
+    'Total number of the main multi-tenant config file reloads.',
+)
+
+mt_config_reload_errors = registry.new_counter(
+    'mt_config_reload_errors_total',
+    'Total number of the main multi-tenant config file reloads.',
+)
+
+mt_tenant_successful_actions = registry.new_labeled_counter(
+    'mt_tenant_successful_actions_total',
```

Comment:
Let's break those down to separate metrics for add/remove/reload

Reply:
roger - and maybe instead of `successful_add`, I should just track the total number of add/remove/reload attempts, as we'll track the failures anyways

Reply:
Sure